.idea/workspace.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Similar to the above comment here^ (If there is any significance, please let me know :see_no_evil:)

Reply:
thanks for spotting this, this is from my IDE (Goland) and should go to `.gitignore`